src/plane-detector.js

```diff
@@ -0,0 +1,11 @@
+class PlaneDetector {
+    static setNativePlaneDetector(planeDetector) {
+        PlaneDetector.nativePlaneDetector = planeDetector
+    }
+
+    static instance() {
```

Comment:
Does it mean we will have to write `PlaneDetector.instance` to call any native plane detector's function?
Can we expose all native plane detector API in here, as static members/functions?

Reply:
No it doesn't - this call returns instance of PlaneDetector.
I don't think exposing is needed in this case PlaneDetector.instance returns NativePlaneDetector which implements common API.